test/isolation/expected/dropchunks_race.out

```diff
@@ -0,0 +1,22 @@
+Parsed test spec with 3 sessions
+
+starting permutation: s1a s2a s3a s3b
+debug_waitpoint_enable
+
+               
+step s1a: SELECT COUNT(*) FROM drop_chunks('dropchunks_race_t1', TIMESTAMPTZ '2020-03-01'); <waiting ...>
+step s2a: SELECT COUNT(*) FROM drop_chunks('dropchunks_race_t1', TIMESTAMPTZ '2020-03-01'); <waiting ...>
+step s3a: SELECT debug_waitpoint_release('drop_chunks_chunks_found');
+debug_waitpoint_release
+
+               
+step s1a: <... completed>
+count          
+
+1              
+step s2a: <... completed>
+error in steps s3a s1a s2a: ERROR:  dimension slice 222 locked by other transaction
```

Comment:
Why do we fail here as opposed to just reading the latest tuple? For `table_tuple_lock` there's an option `TUPLE_LOCK_FLAG_FIND_LAST_VERSION` that I believe would follow the update chain and lock the latest version of the tuple.

Also, this error could be more user friendly. I think it should say something like: `"could not drop chunk <foo> since it was concurrently updated"`

Reply:
I added the `TUPLE_LOCK_FLAG_FIND_LAST_VERSION` but that does not seem to change anything regarding the failure. Still, the flag makes sense, so kept it. Reworded the error message, but chunk name is not available at a reasonable level so not printing it.